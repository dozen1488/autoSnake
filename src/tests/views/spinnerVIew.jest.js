/* eslint-env jest */

import React from "react";
import { mount, configure } from "enzyme";
import Adapter from "enzyme-adapter-react-16";
import toJson from "enzyme-to-json";

import SpinnerRender from "../../application/views/spinnerView";

configure({ adapter: new Adapter() });

describe("Spinner renders correctly", function() {
    it("equals with snapshot", function() {
        expect(
            toJson(mount(<SpinnerRender/>))
        ).toMatchSnapshot();
    });
});
