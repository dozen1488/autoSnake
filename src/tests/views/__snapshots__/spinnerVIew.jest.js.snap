// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Spinner renders correctly equals with snapshot 1`] = `
<renderSpinner>
  <div
    className="fulfilling-bouncing-circle-spinner"
  >
    <div
      className="circle"
    />
    <div
      className="orbit"
    />
  </div>
</renderSpinner>
`;
